packages/cxl-ui/scss/global/cxl-card-grid.scss

```diff
@@ -0,0 +1,7 @@
+cxl-card-grid {
+  cxl-light-card,
+  cxl-course-card {
+    width: 100%;
+    max-width: none;
+  }
+}
```

Comment:
Same problem. By styling the component `:host` directly, we avoid unnecessary complexity with global stylesheets. We should only use global styles when there's no other way.